pages.fr/common/zoxide.md

```diff
@@ -0,0 +1,29 @@
+# zoxide
+
+> Garde une trace des répertoires les plus utilisés.
+> Utilise un algorithme de classement pour identifier le meilleur résultat.
+> Plus d'informations : <https://github.com/ajeetdsouza/zoxide>.
+
+- Aller au répertoire avec le meilleur classement qui contient "foo" dans son nom :
+
+`zoxide query {{foo}}`
+
+- Aller au répertoire avec le meilleur classement qui contient "foo" et "bar" dans son nom ::
+
+`zoxide query {{foo}} {{bar}}`
+
+- Démarre une recherche de répertoire interactive (requires `fzf`) :
+
+`zoxide query --interactive`
+
+- Ajoute un répertoire ou incrémente son classement :
+
+`zoxide add {{chemin/du/répertoire}}`
```

Comment:
Could you please fix the `path/to/dir` to `path/to/directory` in https://github.com/tldr-pages/tldr/blob/master/pages/common/zoxide.md?

Reply:
@navarroaxel Sure, do you want me to do the same with https://github.com/tldr-pages/tldr/blob/master/pages/common/zip.md?

Reply:
Oh, I didn't catch that. Yep, but in a new PR.